### DISCUSSION TITLE
Using issue forms with GitHub Actions

### DISCUSSION THREAD

**Discussion by andyfeller** [community/community#6838] [Projects and Issues]
Issue forms is exciting because of the potential combining it with GitHub Actions.  Thankfully, I've been able to use @peter-murray and https://github.com/peter-murray/issue-forms-body-parser to build issue-driven workflows, however I would love to see whether native support for this is on the roadmap and whether it can be down without the delimiters.

**Comment by Pike** [comment]
Along the same lines, is there a specified route from input to markdown? That'd be really helpful when trying to do integration tests.

**Comment by onnenon** [comment]
Having used this at scale for a company, it would be nice to get some sort of structured data back for the issue. Having to parse values for fields of out the markdown is still fragile. Having some sort of JSON of the issue form data available would make issue driven workflows amazing.

**Comment by andyfeller** [comment]
In the mean time, huge shout out to @PatrickHeneise for his work around https://github.com/zentered/issue-forms-body-parser which offers a clean way to parse and leverage issue form data from issue body.  Totally check it out!

**Reply by mloskot** [reply]
Here is another very cool parser example by @stefanbuck which takes a big different approach of parsing driven by the issue form template itself https://github.com/stefanbuck/github-issue-parser
